src/redux/actions/role-actions.js

```diff
@@ -93,6 +93,29 @@ export const updateRole = (roleId, data, useCustomAccess) => ({
   },
 });
 
+export const patchRole = (roleId, data) => ({
+  type: ActionTypes.PATCH_ROLE,
+  payload: RoleHelper.patchRole(roleId, data),
+  meta: {
+    notifications: {
+      fulfilled: {
+        variant: 'success',
+        title: 'Success updating role',
+        dismissDelay: 8000,
+        dismissable: false,
+        description: 'The role was updated successfully.',
+      },
+      rejected: {
```

Comment:
I would remove the rejected key. The default error notification will display the API error message.